core/src/main/java/hudson/model/queue/CauseOfBlockage.java

```diff
@@ -45,7 +46,7 @@ public void print(TaskListener listener) {
      * Obtains a simple implementation backed by {@link Localizable}.
      */
     public static CauseOfBlockage fromMessage(@NonNull final Localizable l) {
-        l.getKey(); // null check
+        Objects.requireNonNull(l);
```

Comment:
A more direct way of expressing the same concept that also aids the static analyzer.